src/notification/model.rs

```diff
@@ -486,4 +596,49 @@ mod tests {
             }
         }
     }
+
+    async fn create_new_test_user<'a, E>(executor: E) -> Result<i32, sqlx::Error>
+    where
+        E: Executor<'a>,
+    {
+        let mut transaction = executor.begin().await?;
+
+        sqlx::query!(
+            r#"
+                INSERT INTO uuid (trashed, discriminator) VALUES (0, "user")
+            "#
+        )
+        .execute(&mut transaction)
+        .await?;
+
+        let new_user_id = sqlx::query!("SELECT LAST_INSERT_ID() as id FROM uuid")
+            .fetch_one(&mut transaction)
+            .await?
+            .id as i32;
+
+        sqlx::query!(
+            r#"
+                INSERT INTO user (id, username, email, password, token)
+                VALUES (?, ?, ?, "", ?)
+            "#,
+            new_user_id,
+            random_string(10),
+            random_string(10),
+            random_string(10)
```

Comment:
Since `username` and `email` is set to be unique in the MySQL having default values would not work when create_test_user() is called twice or when a transactions was saved in the database (In debugging I put a transaction.commit() at the end of the test and checked the changed database values to find bugs in my code). Having a way to produce unique values here would be great (can be deterministic, though). Do you have an idea?